main.css

```diff
@@ -21,26 +21,202 @@ time, mark, audio, video {
+	font-family: "Arial";
+	text-align: center;
 }
 
 /*********   header   **********/
 header {
-
+	width: 100%;
+	display: inline-block;
+	color: white;
+	background-color: #BD4D4D;
+	font-weight: bold;
+	font-size: 25px;
+	height: 70px;
+	text-align: center;
+	letter-spacing: .5px;
 }
 
 /*********   article   **********/
 article {
-
+	padding-bottom: 71px;
+	position: absolute;
 }
 
 /*********   section   **********/
 section {
+	padding: 2em;
+	padding-bottom: 50px;
+}
 
+/*********   footer   **********/
+footer {
+	background-color: #689CD9;
+	position: fixed;
+	height: 75px;
+	width: 100%;
+	bottom: 0px;
 }
 
 /*********   nav   **********/
 
-nav {
+nav {    
+   	/*display: inline-block;*/
+   	text-align: center;
+   	display: flex;
+   	height: 100%;
+   	align-items: center;
+}
+
+
+img {
+	max-width: 100%;
+	max-height: 100%;
+}
+
+.top-right {
+    height: 50px;
+    width: 50px;
+    position: absolute;
+    right: 10px;
+    top: 10px;
+}
+
+.dropdown {
+	float: left;
+	text-align: center;
+	position: absolute;
+	top: 17px;
+	left: 15px;
+	
+}
+
+.dropbtn {
+	height: 25px;
+	width: 100px;
+	font-size: 15px;
+	border-radius: 6px;
+	border: 1px solid;
+}
+
+.searchBox-button {
+	font-size: 30;
+
+}
+
+.nav-button {
+	margin: 70px;
+	padding: 0%;
+	width: 25%;
+	height: 40%;
+	border-radius: 6px;
+	border: .5px solid;
+	font-size: 15px;
+	text-align: center;
+}
+
+.nav-button:visited {
+	color: #ffffff;
+}
+
+.other-pokemon {
+	display: inline-block;
+	width: 15%;
+	background-color: #ffffff;
+	margin-right: -4px;
+	margin-top: -4.5px;
+}
+
+.other-pokemon img {
+    width: 100%;
+	height: 150px;
+	object-fit: contain;
+	border: 1px solid;
+	margin-right: 0px;
+}
+
+.overview img {
+	width: 150px;
+	height: 150px;
+	padding-left: 150px;
+}
+
+.overview h2 {
+	border: 1px black solid;
+	color: white;
+	background-color: #689CD9;
+	font-size: 25px;
+    padding-left: 30px;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    display: block;  
+    width: 50%;
+    text-align: left;
+}
+
+.overview ul {
+	border: 1px black solid;
+	color: black;
+	background-color: #ffffff;
+	font-size: 20px;
+	padding-left: 30px;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    display: block;  
+    width: 50%;
+    left: 800px;
+    text-align: left;
+
+}
+
+.overview p {
+	border: 10px #BD4D4D solid;
+	border-radius: 15px;
+	color: black;
+	background-color: #ffffff;
+	font-size: 20px;
+	display: block;
+	width: 80%;
+	padding: 30px;
+	margin-top: 50px;
+	text-align: center;
+	line-height: 1.5;
+}
+
+#pokeball-icon {
+	position: relative;
+}
 
+#site-title {
+	text-align: center;
+	padding: 19px;
+}
+
+#search-box {
+	padding: 30px;
+	text-align: center;
+	/*font-size: 50px;*/
+}
+
+input[type="text"]
+{
+    font-size: 20px;
+    padding-left: 20px;
+    padding-top: 5px;
+    padding-bottom: 5px;
+}
+
+/******************* ul *******************/ 
+/*Borrowed from :https://stackoverflow.com/questions/3200249/html-list-style-type-dash*/
```

Comment:
thanks for letting us know! good use of understanding other people's code and incorporating